### ISSUE TITLE
Remapping Up to e.g. Right trigger

### ISSUE THREAD

**Issue by dfilskov** [Mazamars312/Analogue-Amiga#21]
Great work on the latest update - it's so useful with keyboard!

Please consider making it possible to let the right trigger button and/or one of the four fire buttons work as joystick Up - that'll be reeally useful for many gametypes :)

**Comment by Mazamars312** [comment]
Not really, as the joystick input does not get modified by the MPU core. I think there is a change in wdload that can do this.

**Comment by dfilskov** [comment]
Thank you for looking into it!

I've noticed that other cores list the input as the original devices - like

Joystick port 1 up
Joystick port 1 down
(lert and right)
...
Fire button 1
Fire button 2

and/or

CD32 ctrl 1 up
CD32 ctrl 1 down
... (and its four fire buttons)

Whereas your core list the Analogue Pocket's controls like

D-pad up
D-pad down
...
Left shoulder
Right shoulder

Perhaps it'd be easier for us users if we could see the original device names instead of mapping e.g. 'Left shoulder button' to 'Left shoulder button' - that's somewhat confusing :)

**Comment by dfilskov** [comment]
Here's how the Atari core controls / remapping looks.

Perhaps this core could do the same but with Amiga devices (jlystick

**Comment by dfilskov** [comment]
Here's how the Atari core controls / remapping looks.

![IMG_20230727_211620~3](https://github.com/Mazamars312/Analogue-Amiga/assets/15096058/1ae2844b-ff8d-43e6-8412-e38fd7574977)


Perhaps this core could do the same but with Amiga devices (joystick, mouse, and some keys)

**Comment by MHzBurglar** [comment]
@Mazamars312 

Is it possible to create an input entry that maps a Pocket button to a specific Amiga key code/event?  It looks like there is a specific key code for "Port 0 Joystick Up" according to this: https://wiki.amigaos.net/wiki/Keymap_Library#Raw_Key_Table

Ideally, we would like the d-pad to still function as the joystick x/y axis as normal, but _also_ have the Pocket's A button (or whatever the user defines) specifically send the "joystick up" event.
